Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.messages.in

```diff
@@ -64,6 +64,10 @@ messages -> WebSWServerConnection NotRefCounted {
     RetrieveRecordResponse(WebCore::BackgroundFetchRecordIdentifier recordIdentifier) -> (Expected<WebCore::ResourceResponse, WebCore::ExceptionData> result)
     RetrieveRecordResponseBody(WebCore::BackgroundFetchRecordIdentifier recordIdentifier, WebKit::RetrieveRecordResponseBodyCallbackIdentifier callbackIdentifier)
 
+    AddCookieChangeSubscriptions(WebCore::ServiceWorkerRegistrationIdentifier identifier, Vector<WebCore::CookieChangeSubscription> subscriptions) -> (std::optional<WebCore::ExceptionData> result)
```

Comment:
All these new IPC end points likely need to be guarded behind a `[EnabledBy=X]` for security.

Reply:
Should I used both `CookieStoreAPIEnabled` and `CookieStoreManagerEnabled` flags or just `CookieStoreManagerEnabled` ?

Reply:
I used just `CookieStoreManagerEnabled` since these functions are only being called from `CookieStoreManager`